Task2/CalculatingWithFunctions.js

```diff
@@ -0,0 +1,56 @@
+// 1-http://www.codewars.com/kata/calculating-with-functions
+function zero() {
+    return arguments.length ? arguments[0](0) : 0;
```

Comment:
Do proper indentation, it will increase the readability.
Here the first line is starting for start of the page. all other method are not starting from the start of the page.
i would cause problems for developers when we implement a huge code and if it not properly indented.